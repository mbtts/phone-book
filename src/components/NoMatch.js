import React from "react";

const NoMatch = () => {
  return <p>Page cannot be found</p>;
};

export default NoMatch;
