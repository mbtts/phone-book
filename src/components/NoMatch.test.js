import NoMatch from "./NoMatch";
import React from "react";

describe("No match component", () => {
  it("should render initial state", () => {
    const wrapper = shallow(<NoMatch />);
    expect(wrapper).toMatchSnapshot();
  });
});
