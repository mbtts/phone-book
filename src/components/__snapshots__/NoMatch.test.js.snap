// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No match component should render initial state 1`] = `
<p>
  Page cannot be found
</p>
`;
